### ISSUE TITLE
A question about anniversaries and another thing.

### ISSUE THREAD

**Issue by MaximumRustler** [Monika-After-Story/MonikaModDev#1403]
So I started playing this mod on February 14 and I got the 1 week anniversary but didn't get the 1 month one last month, I got the White Day event though. I noticed in the patch notes that this was a bug and it was fixed, so I was wondering when I get to the next anniversary day would I get the right dialog or would I get the 1 month dialog for some reason?

Also it seems like the fast food topic should be a non repeating one because it starts out with Monika saying "Hm? Do I like fast food?", the way it's worded sounds more like a question the player would ask her.

And finally, thank you guys for creating this mod. It really is helping me get through a pretty terrible time in my life.

**Comment by Tiler17** [comment]
Huh. There's dialogue for one month? I didn't get that either. I downloaded this a day before you, on February 13th.

**Comment by MaximumRustler** [comment]
Yeah, neither did I until I read the patch notes.

**Comment by aldoram5** [comment]
Yeah there was an issue with the anniversary events, you can see the missing events if you change the date on your computer to the corresponding anniversary date. So in your case MaximumRustler you should change the date to March 14, just in case backup your persistent file, I don't think there would be any issue since I did that for my own game and nothing wrong has happened, but it's always a good practice to backup that file.
As for your question regarding the fast food topic yeah it is kinda strange that it is random, maybe we'll change that, it depends on what the others think about it.

**Comment by MaximumRustler** [comment]
Thanks for the info. So changing the date to march 14 and back wont have any strange effects? And yeah I've been backing up my persistent file every time I exit the game just in case.
